### PR TITLE
feature/mx-1347 link cruft template

### PR DESCRIPTION
**Pull request by cutoffthetop**
# Added
- pull request template
- CHANGELOG file
- cruft template link

# Changes
- harmonized boilerplate

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Changelog
```
Comment (rababerladuseladim) [inline CHANGELOG.md:1]: Please add your changes.

File: README.md [hunk 2]
```diff
@@ -2,8 +2,8 @@
 
 Metadata editor web application.
 
-[![linting](https://github.com/robert-koch-institut/mex-common/actions/workflows/linting.yml/badge.svg)](https://github.com/robert-koch-institut/mex-common/actions/workflows/linting.yml)
-[![cve-scan](https://github.com/robert-koch-institut/mex-common/actions/workflows/cve-scan.yml/badge.svg)](https://github.com/robert-koch-institut/mex-common/actions/workflows/cve-scan.yml)
+[![linting](https://github.com/robert-koch-institut/mex-editor/actions/workflows/linting.yml/badge.svg)](https://github.com/robert-koch-institut/mex-editor/actions/workflows/linting.yml)
```
Comment (rababerladuseladim) [inline README.md:5]: Oh...

**Review by rababerladuseladim — CHANGES_REQUESTED** [review]
Miniminor stuff.

File: README.md (hunk 2 above)
Comment (cutoffthetop) [reply README.md:5]: yeah 😅

File: CHANGELOG.md (hunk 1 above)
Comment (cutoffthetop) [reply CHANGELOG.md:1]: done 17c22d4

**Review by rababerladuseladim — APPROVED** [review]

**Pull request merged**